app/assets/stylesheets/forms.scss

```diff
@@ -38,18 +38,12 @@ section.sign-up {
   }
 }
 
-.edit_team, .edit_namespace {
-  textarea {
-    max-height: 600px;
-    min-height: initial;
-    max-width: 100%;
-  }
-}
-
-#new-namespace-form, #new-team-form {
+.edit_team, .edit_namespace, #new-namespace-form, #new-team-form, #new-comment-form {
```

Comment:
Instead of doing this, why don't you create a class named like `.fixed-textarea`, and apply this class to every textarea in which we want to put these limits (in our case to every one of them). This way this would work for whenever we add a new form and we forget to add it here.